boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
What is the difference between

```js
$tw.loadTiddlersBrowser(finishStartup);
```

and

```js
$tw.loadTiddlersBrowser();
finishStartup();
```

In this use case? Not understand the need for the call back deep in the system when you could simply wrap it:

```js
function bootTiddlyAsync(callback) {
  $tw.loadTiddlersBrowser();
  callback();
}
```